### ISSUE TITLE
[HELP] How to run 1.5.0

### ISSUE THREAD

**Issue by MarcoRavich** [protyposis/AudioAlign#9]
Hi there, we're excited for this new version but if we try to run AudioAlign.exe it just draws a windows and suddently closes: any hint ?

**Comment by protyposis** [comment]
Do you have the .NET ~SDK~ Windows desktop runtime 6 installed? What OS and OS version do you run this on?

**Comment by MarcoRavich** [comment]
Ok, runs now (didn't get that SDKs are needed, we've desktop .NET instead).

More feedbacks will come.

note: as many other sws does, it would be useful to give en error message that specifies the needs.

**Comment by dotbanana** [comment]
I try to run this under wine but it unfortunately doesn't work. 
Windows Desktop Runtime 6 is installed. 
Can you give a list with all the dependencies?

**Comment by MarcoRavich** [comment]
> I try to run this under wine but it unfortunately doesn't work. Windows Desktop Runtime 6 is installed. Can you give a list with all the dependencies?
>
Since AA is Windows-specific app/lib dunno if you can run under Wine...

...I've already opened [this Issue @ Aurio git](https://github.com/protyposis/Aurio/issues/36) to ask for a C++ "port" in order to hopefully obtain a platform-indipendent version.

Anyway, if you can't succed with AudioAling consider other open resources - I've collected for the [HyMPS project](https://forart.it/HyMPS) (under [AUDIO section](https://github.com/forart/HyMPS#- ) \ [Treatments page](https://github.com/forart/HyMPS/blob/main/Audio/Treatments.md#-- ) \ [Alignmentsynch](https://github.com/forart/HyMPS/blob/main/Audio/Treatments.md#alignmentsynch-)) - too.

**Comment by dotbanana** [comment]
Thanks for your answer. 
Yeah this would be awesome, this tool is really one of its kind.  
I've tinkered around for a couple of hours and found out following regarding wine if someone other looks this up: 

Bottles doesn't work at all with the integrated dependency/library-downloader.  
I've got it running with Lutris.  
Here a quick how-to:
In Lutris in the burgermenu go to "Add Games", chose "add locally installed game", choose AudioAlign.exe, Starter: Wine. 
Wine version: System (10.12) Staging
Check "Use winetricks of system". 
Activate Virtual Desktop (necessary to get the GUI working) in options. 
When you try to open it this way he will complain about missing net Desktop 6. 
Follow the link to the microsoft website and download exactly the version he offers you there. 
Open in Lutris at the bottom Winetricks. 
There you go to "Install Programm", then you click "abort/cancel", then you go to "Run arbitrary executable" and choose the installer you downloaded from microsoft and install it into the wine-prefix through winetricks. 
I've added some more audio-libs through the winetricks-manager, tho i don't know if this was necessary. 
But it runs now with occasional crashes when zooming in. And i am sure with a little bit more tinkering this could be also fixed :)
